### ISSUE TITLE
nil for default argument value is not rendered in docs

### ISSUE THREAD

**Issue by e-kayrakli** [chapel-lang/chapel#13531]
The default value is simply dropped and shows like "someArgument="

In my use case the type of the argument is locale and I have tried

- `arg=nil:locale` ("locale" is printed fine)
- `arg:locale=nil` ("locale" is printed fine)
- `arg=nil`

none seemed to show `nil` in the method signature.

**Comment by cassella** [comment]
This affects the documentation of a few HDF5 functions:

https://chapel-lang.org/docs/modules/packages/HDF5.html#HDF5.readAllNamedHDF5Files
https://chapel-lang.org/docs/modules/packages/HDF5.html#HDF5.readNamedHDF5FilesInto1DArrayInt
https://chapel-lang.org/docs/modules/packages/HDF5.html#HDF5.hdf5ReadChunks

**Comment by lydia-duncan** [comment]
Good catch!  I suspect this won't be too hard to fix - I'll add it to my next sprint cycle

**Comment by lydia-duncan** [comment]
Actually, I think @cassella  did in #17121.  Regardless, I think it can be closed
